### PR TITLE
update ce_bgp to fix bugs

### PR DESCRIPTION
**Pull request by yanzhangi**
##### SUMMARY

Modify an error: When the vrf_name parameter is in the module and the configuration is repeatedly sent to the device, the module displays change = True

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ansible/modules/network/cloudengine/ce_bgp.py
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/network/cloudengine/ce_bgp.py [hunk 1]
```diff
@@ -1675,14 +1675,26 @@ def merge_bgp_instance(self, **kwargs):
...
         """ create_bgp_instance """
 
         module = kwargs["module"]
         conf_str = CE_CREATE_BGP_INSTANCE_HEADER
 
+        cmds = []
+
         vrf_name = module.params['vrf_name']
         if vrf_name:
+            if vrfName == "_public_":
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_bgp.py]: vrfName ?

**Review by yuandongx — CHANGES_REQUESTED** [review]

**Comment by hwDCN** [comment]
LGMT

**Comment by hwDCN** [comment]
+1

**Review by yuandongx — APPROVED** [review]
+1

**Pull request merged**